data/json/flags.json

```
@@ -2333,6 +2333,18 @@
     "info": "A medium-sized gasmask filter.",
     "name": "gas mask cartridge"
   },
+  {
+    "id": "PANORAMIC_OUTSERT",
+    "type": "json_flag",
+    "info": "This item is meant to be externally attached to the eyepiece of a single-Lenz gas mask.",
```

Unless this has some other meaning?
```suggestion
    "info": "This item is meant to be externally attached to the eyepiece of a single-lens gas mask.",
```